Submodule draft ietf rats corim

nedmsmith opened the pull request:
Cutting over to updated corim repo. Involves removing previous submodule.

bhenning10 commented on comid-extensions.cddl:
```diff
@@ -2,58 +2,18 @@
-  comid.operational-flag-runtime-meas => bool
-  comid.operational-flag-immutable-meas => bool
-  comid.operational-flag-tcb-meas => bool
+  &(runtime-meas: 6) => bool
```
Question: Does this change make the flags-map-extension a choice between these three options (6,7,8)?

nedmsmith replied:
All extensions are treated as optional since the extension is technically optional. If a ? is added it throws a syntax error. I think this is incorrect behavior personally. The human readable representation is also a goal of CDDL and even if the extension made it optional automatically, adding ? should be treated as idempotent.

nedmsmith replied:
However, when I removed one of the flags from the test, the test failed. And when I added '?' to the cddl the test passed.

bhenning10 replied:
Ok, that makes sense

nedmsmith commented on Makefile:
```diff
@@ -2,15 +2,73 @@
 
 CORIM_DEPS :=
-CORIM_DEPS += $(CORIM_DIR)/comid-autogen.cddl
+#CORIM_DEPS += $(CORIM_DIR)/comid-autogen.cddl
```
Yes. It should be removed.

bhenning10 reviewed (commented):
I have a couple of questions/comments

nedmsmith commented:
> I have a couple of questions/comments

I added '?' to flags extension map items and added a test for optional flags via extension point and removed spurious comment in Makefile.
